DB privs to run scripts in "Identifying and Removing Corrupt Data" article. (#82)
exasol/public-knowledgebase: Database-Features/removing-corrupt-data.md

@/tmp/check_all_columns.sql;
```

> To execute the file, run `exaplus -u <username> -p <passwd> -c <connection string> -f corrupt_data_check.sql`. DBA role would be sufficient for DB user to execute this script, see [Roles](https://docs.exasol.com/db/latest/database_concepts/database_users_roles.htm#Roles).

Reviewer: ```suggestion
To execute the file, run `exaplus -u <username> -p <passwd> -c <connection string> -f corrupt_data_check.sql`. The user running this script requires the DBA role, see [Roles](https://docs.exasol.com/db/latest/database_concepts/database_users_roles.htm#Roles) for more information.
```

> To execute the file, run `exaplus -u <username> -p <passwd> -c <connection string> -f corrupt_data_check.sql`. Database user requires SELECT ANY DICTIONARY system privilege for executing this script, see [Details on Rights Management](https://docs.exasol.com/db/latest/database_concepts/privileges/details_rights_management.htm).

Reviewer: @allipatev the user also needs the permission to SELECT every table in the database. I would rather state that the user must be a DBA for simplification.

Author: Agree, it seemed obvious to me yesterday ...

@/tmp/enforce_indexes.sql;
```

> To execute the file, run `exaplus -u <username> -p <passwd> -c <connection string> -f recreate_indexes.sql`. DBA role would be sufficient for DB user to execute this script, see [Roles](https://docs.exasol.com/db/latest/database_concepts/database_users_roles.htm#Roles). You can view `/tmp/index_logs.log` or auditing in the database to ensure that all queries executed successfully.

Reviewer: ```suggestion
To execute the file, run `exaplus -u <username> -p <passwd> -c <connection string> -f recreate_indexes.sql`. The user running this script requires the DBA role, see [Roles](https://docs.exasol.com/db/latest/database_concepts/database_users_roles.htm#Roles) for more information. You can view `/tmp/index_logs.log` or auditing in the database to ensure that all queries executed successfully.
```

> To execute the file, run `exaplus -u <username> -p <passwd> -c <connection string> -f recreate_indexes.sql`. Database user requires privileges to create and drop indices to run this script. DBA role would be sufficient, see [Roles](https://docs.exasol.com/db/latest/database_concepts/database_users_roles.htm#Roles). You can view `/tmp/index_logs.log` or auditing in the database to ensure that all queries executed successfully.

Reviewer: For simplification, I would also put here that it requres DBA privileges, no need to mention something specific about creating and dropping indices
